config/main.py

```diff
@@ -524,6 +524,25 @@ def load_minigraph():
     click.echo("Please note setting loaded from minigraph will be lost after system reboot. To preserve setting, run `config save`.")
 
 
+#
+# 'hostname' command
+#
+@config.command('hostname')
+@click.argument('new_hostname', metavar='<new_hostname>', required=True)
+def hostname(new_hostname):
+    """Change Hostname on a SONiC device without impacting the traffic."""
```

Comment:
@samaity: Since the docstrings get truncated when displayed in the help message, I think you should shorten this message so that more of it is displayed. The phrase "on a SONiC device" is unnecessary, as the CLI is only used on SONiC devices.

`"""Change device hostname without impacting traffic."""`